spec/requests/api/moves_controller_index_v2_spec.rb

```diff
@@ -23,7 +23,7 @@
     let!(:moves) { create_list :move, 2 }
 
     it_behaves_like 'an endpoint that responds with success 200' do
-      before { do_get }
+      before { do_get('?include=from_location') }
```

Comment:
We have a shared schema for `Move` which lists `from_location` as a required relationship, so it needs to be explicitly included here so that it gets rendered conditionally and  the response validates against the schema.